vsintegration/Vsix/VisualFSharpFull/Source.extension.vsixmanifest

```diff
@@ -9,7 +9,9 @@
     <MoreInfo>https://docs.microsoft.com/en-us/dotnet/articles/fsharp/</MoreInfo>
   </Metadata>
   <Installation Experimental="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[|%CurrentProject%;GetVSGeneralVersion|,)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[|%CurrentProject%;GetVSGeneralVersion|,)">
```

Comment:
Does the Change from Community to Pro mean anything?

Reply:
no